drivers/flash/flash_ipuc/flash_ipuc.c

```diff
@@ -560,6 +647,9 @@ static struct device *flash_cache_ipuc(uintptr_t min_address, uintptr_t *ipuc_ad
 			(uintptr_t)ipuc_size);
 		return NULL;
 	}
+#if defined(CONFIG_SOC_FLASH_NRF_RADIO_SYNC_RPC_HOST)
+	nrf_flash_sync_init();
```

Comment:
I think `flash_component_ipuc_create` might be a better place (or at least - it should be added there too) - if the application does not enable `DFU_CACHE_RW` than this function is not called at the sysinit level.

Reply:
This is fixed